packages/kbn-securitysolution-endpoint-exceptions-common/api/find_endpoint_list_item/find_endpoint_list_item.gen.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Find endpoint list items API endpoint
+ *   version: 2023-10-31
+ */
+
+import { z } from 'zod';
+
+import { NonEmptyString } from '@kbn/openapi-common/schemas/primitives.gen';
+import { EndpointListItem } from '../model/endpoint_list_common.gen';
+
+export type FindEndpointListItemsFilter = z.infer<typeof FindEndpointListItemsFilter>;
+export const FindEndpointListItemsFilter = NonEmptyString;
+
+export type FindEndpointListItemsRequestQuery = z.infer<typeof FindEndpointListItemsRequestQuery>;
+export const FindEndpointListItemsRequestQuery = z.object({
+  /** 
+      * Filters the returned results according to the value of the specified field,
+using the `<field name>:<field value>` syntax.
```

Comment:
Nit: the spacing on this multiline comment is a little wonky. Any idea what happened here?